crates/red_knot_python_semantic/src/types.rs

```diff
@@ -362,15 +362,15 @@ impl<'db> Type<'db> {
         matches!(self, Type::Todo)
     }
 
-    pub const fn into_class_literal_type(self) -> Option<ClassType<'db>> {
+    pub const fn into_class_literal_type(self) -> Option<ClassLiteralType<'db>> {
```

Comment:
Maybe name this consistently (similar for module/function) to match the names of `expect_class_literal` and `is_class_literal`?
```suggestion
    pub const fn into_class_literal(self) -> Option<ClassLiteralType<'db>> {
```
Or the other way around (add `_type` suffix consistently)?

Reply:
Removed all `*_type` suffixes in https://github.com/astral-sh/ruff/pull/14108/commits/a97d183e7cb5a458f288e7ce53bd4b54e8ebb329 👍